datadog/resource_datadog_synthetics_test_.go

```diff
@@ -295,6 +303,44 @@ func syntheticsTestOptions() *schema.Schema {
 	}
 }
 
+func syntheticsTestOptionsList() *schema.Schema {
+	return &schema.Schema{
+		Type:          schema.TypeList,
+		Optional:      true,
+		MaxItems:      1,
+		ConflictsWith: []string{"options"},
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"allow_insecure": {
+					Type:     schema.TypeBool,
+					Optional: true,
+				},
+				"follow_redirects": {
+					Type:     schema.TypeBool,
+					Optional: true,
+				},
+				"tick_every": {
+					Type:     schema.TypeInt,
+					Optional: true,
+				},
+				"accept_self_signed": {
+					Type:     schema.TypeBool,
+					Optional: true,
+				},
+				"min_location_failed": {
+					Type:     schema.TypeInt,
+					Default:  1,
+					Optional: true,
+				},
+				"min_failure_duration": {
+					Type:     schema.TypeInt,
+					Optional: true,
```

Comment:
Don't forget to add the 2 new retry option there.

Reply:
I thought I should have this PR just to add the new `options_list` then have a [second PR](https://github.com/DataDog/terraform-provider-datadog/pull/625) to add the new options

Reply:
Hum ok, then maybe not deprecate `options` there? We should be able to release at any point.

Reply:
Oh yes I see. I can remove deprecate `options` in the second PR.

Reply:
The other way around? Remove `Deprecated` from here, and add it in the second PR that has feature parity.

Reply:
Yes that's what I meant, sorry I wasn't clear